src/test/java/org/openrewrite/java/migrate/javax/RemoveTemporalAnnotationTest.java

```diff
@@ -0,0 +1,118 @@
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.javax;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class RemoveTemporalAnnotationTest implements RewriteTest {
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.parser(JavaParser.fromJavaVersion().classpathFromResources(new InMemoryExecutionContext(), "javax.persistence-api-2.2"))
+          .recipe(new RemoveTemporalAnnotation());
+    }
+
+    @Test
+    @DocumentExample
+    void removeTemporalAnnotation() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import javax.persistence.Temporal;
+              import javax.persistence.TemporalType;
+              import java.sql.Date;
+              import java.sql.Time;
+              import java.sql.Timestamp;
+
+              public class TemporalDates {
+                  @Temporal(TemporalType.DATE)
+                  private Date dateDate;
+
+                  @Temporal(TemporalType.TIME)
+                  private Date dateTime;
+
+                  @Temporal(TemporalType.DATE)
+                  private Time timeDate;
+
+                  @Temporal(TemporalType.TIME)
+                  private java.sql.Time timeTime;
+
+                  @Temporal(TemporalType.TIMESTAMP)
+                  private java.sql.Time timeTimestamp;
+
+                  @Temporal(TemporalType.TIMESTAMP)
+                  private java.sql.Timestamp timestampTimestamp;
+              }
+              """,
+            """
+              import javax.persistence.Temporal;
+              import javax.persistence.TemporalType;
+              import java.sql.Date;
+              import java.sql.Time;
+              import java.sql.Timestamp;
+
+              public class TemporalDates {
+                  private Date dateDate;
+
+                  private Date dateTime;
+
+                  private Time timeDate;
+
+                  private java.sql.Time timeTime;
+
+                  private java.sql.Time timeTimestamp;
+
+                  private java.sql.Timestamp timestampTimestamp;
+              }
+              """
+          )
+        );
+    }
+
```

Comment:
It probably makes sense to have a test that uses Date, Time, and Timestamp classes from other packages such as `java.util.Date`, `java.security.Timestamp`, `org.omg.Security.Time` to make it clear those are valid cases

Reply:
Yeah the idea behind the tests using Date, Time, and Timestamp classes from other packages was to make it clear the recipe should only apply to `java.util.Date`, `java.security.Timestamp`, `org.omg.Security.Time`. I see I had a typo in my original comment.